facia-tool/app/controllers/FaciaContentApiProxy.scala

```diff
@@ -65,11 +84,11 @@ object FaciaContentApiProxy extends Controller with Logging with ExecutionContex
     val ophanApiHost = Configuration.ophanApi.host.get
     val ophanKey = Configuration.ophanApi.key.map(key => s"&api-key=$key").getOrElse("")
 
-    val url = s"$ophanApiHost/$path?$queryString&$paths&ophanKey"
+    val url = s"$ophanApiHost/$path?$queryString&$paths&$ophanKey"
```

Comment:
Do you know what's going to happen now that this will get put in? Maybe speak to ophan about it.